packages/uniforms-bootstrap5/src/AutoField.tsx

```diff
@@ -13,7 +14,7 @@ import TextField from './TextField';
 
 const AutoField = createAutoField(props => {
   if (props.component) {
-    return props.component;
+    return props.component as ComponentType<any>;
```

Comment:
Isn't `props.component` already a `ComponentType<any>`?

Btw, there is also a `Component` type that is an alias for
```tsx
/** @internal */
export type Component = ComponentType<any> | ReturnType<typeof connectField>;
```